src/ExadelMentorship.BackgroundApp/DirectExchangeConsumer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System.Text;
+
+namespace ExadelMentorship.BackgroundApp
+{
+    public static class DirectExchangeConsumer
```

Comment:
JFYI. No need to change!

For proof-of-concept, I'll let it be static, but later for real implementation we need it be a proper service in DI.

Reply:
ok